lesson13/script.js

```diff
@@ -0,0 +1,86 @@
+function showLoadingImg(){
+    const div = document.createElement('div');
+    const loadingImg = document.createElement('img');
+    div.id = "js-loading";
+    loadingImg.src = "img/loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    document.body.insertAdjacentElement('afterbegin',div).appendChild(loadingImg);
+}
+
+function removeLoadingImg(){
+    document.getElementById('js-loading').remove();
+}
+
+async function fetchdata(){
+    const endpoint = "https://api.json-generator.com/templates/szdgGQcOLXuk/data?access_token=hu4bc7qh9znx2m8f53mn4mz2hryvdntkavwbw8j0";
+    const response = await fetch(endpoint);
+    if(!response.ok){
+    throw new Error(`${response.status}:${response.text}`);
```

Comment:
MUST
I think I reviewed this one before, but I think I told you by mistake. The correct name is statusText. Sorry...😫
```suggestion
        throw new Error(`${response.status}:${response.statusText}`);
```
[Please check the operation here](https://codesandbox.io/s/js-lesson13-forked-i0evx1?file=/script.js)

Reply:
Thank you for your comment and for preparing the code sandbox!
I will fixed.

Reply:
I fixed.
https://github.com/nakagawamai/js-lesson/pull/15/commits/d42b8309d89e134f6ca7f1f20e6ed1f56af79882

Reply:
Thank you so much!